### DISCUSSION TITLE
June 19 Stands Up

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#300] [Q&A] [answered]
What's the hold up?

**Comment by hollowaykeanho** [comment, marked as answer]
Finalizing all the pages... can be done today. =)

Basically the latest update broke some minor stuffs here and there. It's better to touch up before publish.

**Comment by corygalyna** [comment]
Your ARIA progress?

**Comment by hollowaykeanho** [comment]
pending your reply?

**Comment by corygalyna** [comment]
Oh yeah... my bad. It's a tough assignment though.

**Comment by hollowaykeanho** [comment]
Yeah.. that's why I need your help. Likely a case-study dataset research before we can actually get down to graphics sketching.

**Comment by corygalyna** [comment]
Ok.. Anything else?

**Comment by hollowaykeanho** [comment]
Jean not with you right? Please cool her down... she's was waiting for the release inpatiently.

**Comment by corygalyna** [comment]
Hahaha... you took 4 months to refactor the project. What do you think? Ok.. let's keep our chat in private.
